src/components/ui/Toast/toast.scss

```diff
@@ -0,0 +1,121 @@
+@import "src/styles/scaffold";
+
+[data-fs-toast] {
+  // --------------------------------------------------------
+  // Design Tokens for Toast
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-toast-width                        : calc(100% - (2 * var(--fs-spacing-3)));
+
+  --fs-toast-min-height                   : var(--fs-spacing-10);
+  --fs-toast-padding                      : var(--fs-spacing-1);
+  --fs-toast-margin                       : var(--fs-spacing-3) var(--fs-spacing-3) 0 var(--fs-spacing-3);
+
+  --fs-toast-border-radius                : 8px;
+  --fs-toast-border-width                 : var(--fs-border-width-default);
+  --fs-toast-border-color                 : transparent;
+
+  --fs-toast-shadow                       : var(--fs-shadow);
+  --fs-toast-bkg-color                    : var(--fs-color-neutral-0);
+
+  --fs-toast-transition-timing            : var(--fs-transition-timing);
+  --fs-toast-transition-function          : var(--fs-transition-function);
+
+  --fs-toast-icon-container-width         : var(--fs-spacing-7);
+  --fs-toast-icon-container-height        : var(--fs-toast-icon-container-width);
+  --fs-toast-icon-container-bkg-color     : var(--fs-color-primary-bkg-light);
+  --fs-toast-icon-container-border-radius : var(--fs-border-radius-default);
+
+  --fs-toast-content-padding              : var(--fs-spacing-0) var(--fs-spacing-2);
+
+  --fs-toast-content-title-size           : var(--fs-text-size-body);
+  --fs-toast-content-title-weight         : var(--fs-text-weight-bold);
+  --fs-toast-content-title-line-height    : 1.25;
```

Comment:
I wonder if we can use the following:

```suggestion
  --fs-toast-content-title-line-height    : var(--fs-scale);
```

![CleanShot 2022-05-16 at 19 50 12](https://user-images.githubusercontent.com/381395/168694897-8fc01176-60ec-4635-a5d8-06ae3463d481.png)